fastlane/Fastfile

```diff
@@ -1,18 +1,23 @@
 # frozen_string_literal: true
 
+BUILD_FOLDER = File.join(__dir__, '.build')
+BUILD_LOG_PATH = File.join(BUILD_FOLDER, 'logs')
+DERIVED_DATA_PATH = File.join(BUILD_FOLDER, 'derived-data')
+PROJECT_PATH = File.join(__dir__, 'TracksDemo', 'TracksDemo.xcodeproj')
```

Comment:
I think you'll get "fastlane/TracksDemo/TracksDemo.xcodeproj" as a result, rather than 'TracksDemo/TracksDemo.xcodeproj'?

Reply:
Ooo...okay...